http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -539,61 +539,42 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
           config.deleteVmPollConfig.interval
         ).compile.lastOrError
 
-        continue = for {
-          diskResourceOpt <- controlledResourceQuery
-            .getWsmRecordForRuntime(runtime.id, WsmResourceType.AzureDisk)
-            .transaction
-          _ <- logger
-            .info(ctx.loggingCtx)(
-              s"No disk resource found for delete azure runtime msg $msg. No-op for wsmDao.deleteDisk."
-            )
-            .whenA(diskResourceOpt.isEmpty)
-          deleteDisk = diskResourceOpt.traverse { disk =>
-            wsmDao.deleteDisk(
-              DeleteWsmResourceRequest(
-                msg.workspaceId,
-                disk.resourceId,
-                DeleteControlledAzureResourceRequest(
-                  WsmJobControl(WsmJobId(s"delete-disk-${ctx.traceId.asString.take(10)}"))
-                )
-              ),
-              auth
-            )
-          }.void
-
-          networkResourceOpt <- controlledResourceQuery
-            .getWsmRecordForRuntime(runtime.id, WsmResourceType.AzureNetwork)
-            .transaction
-          _ <- logger
-            .info(ctx.loggingCtx)(
-              s"No network resource found for delete azure runtime msg $msg. No-op for wsmDao.deleteNetworks."
-            )
-            .whenA(networkResourceOpt.isEmpty)
-          deleteNetworks = networkResourceOpt.traverse { network =>
-            wsmDao.deleteNetworks(
-              DeleteWsmResourceRequest(
-                msg.workspaceId,
-                network.resourceId,
-                DeleteControlledAzureResourceRequest(
-                  WsmJobControl(WsmJobId(s"delete-networks-${ctx.traceId.asString.take(10)}"))
-                )
-              ),
-              auth
-            )
-          }.void
+        deleteDiskAction = msg.diskIdToDelete.traverse { _ =>
```

Comment:
nit, can we move this to after `logger.info(ctx.loggingCtx)("runtime is deleted successfully")` line